src/main/java/it/gov/pagopa/payment/service/payment/qrcode/QRCodeUnrelateServiceImpl.java

```diff
@@ -42,12 +42,12 @@ public void unrelateTransaction(String trxCode, String userId) {
             TransactionInProgress trx = authorizationExpiredService.findByTrxCodeAndAuthorizationNotExpired(trxCode.toLowerCase());
 
             if (trx == null) {
-                throw new ClientExceptionNoBody(HttpStatus.NOT_FOUND, "[UNRELATE_TRANSACTION] Cannot find transaction having code: %s".formatted(trxCode));
+                throw new TransactionNotFoundOrExpiredException(ExceptionCode.TRX_NOT_FOUND_OR_EXPIRED, "[UNRELATE_TRANSACTION] Cannot find transaction having code: %s".formatted(trxCode));
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor